crates/bevy_input/src/touch.rs

```diff
@@ -127,11 +127,14 @@ pub enum ForceTouch {
     reflect(Serialize, Deserialize)
 )]
 pub enum TouchPhase {
```

Comment:
One thing that feels weird to me is using the `TouchPhase` from touch.rs in mouse.rs. Open to suggestions if this is a concern.